### PR TITLE
Making wi details module independent

### PR DESCRIPTION
**Pull request by sanbornsen**
- Moved the details slider panel animation and css classes from list view and board view to details view.

### CODE REVIEW

**Comment by sanbornsen** [comment]
[test]

**Comment by nimishamukherjee** [comment]
Work item detail remains under work-item-list?

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 1]
```diff
@@ -331,6 +348,12 @@
     }
   }
 
+  .detail-panel{
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: Add space before {

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 2]
```diff
@@ -331,6 +348,12 @@
...
+    width: $width100;
+    padding: 0 ;
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: remove space before ;.

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 3]
```diff
@@ -331,6 +348,12 @@
...
+    top:0;
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: add space after :.

**Comment by sanbornsen** [comment]
@nimishamukherjee that will be taken out but not in this PR.

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 4]
```diff
@@ -236,6 +236,23 @@
...
+
+//detail panel
+.detail-panel {
+  background: $color-pf-white;
+  position: fixed;
+  left: auto;
+  top: em(34);
+  right: 0;
+  bottom: 0;
+  width: 50%;
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: Convert to em

Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: This is the width of the panel, i.e. half of the page. How to convert % into em? @SMahil any idea?

File: src/app/work-item/work-item-list/work-item-list.component.html [hunk 5]
```diff
@@ -111,6 +111,6 @@ <h4 class="list-type">Backlog</h4>
     </div>
 </div>
 
-<div class="detail-panel" [@slideInOut]="panelState">
+
   <router-outlet></router-outlet>
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-list.component.html]: no need of indentation. and move it to line no. 114.

File: src/app/work-item/work-item-list/work-item-list.component.html [hunk 6]
```diff
@@ -111,6 +111,6 @@ <h4 class="list-type">Backlog</h4>
...
-</div>
+
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-list.component.html]: no need of this newline.

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 7]
```diff
@@ -273,7 +290,7 @@
       background: $color-pf-white;
       list-style: none;
       margin-bottom: 0;
-      padding: 0; 
+      padding: 0;
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: There is a common style 'no-padding' in stylesheets/custom.scss

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss (hunk 4 above)
Comment (SMahil) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: I don't feel we should change it to em as we always want half of the page

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html [hunk 8]
```diff
@@ -1,236 +1,238 @@
...
-                  </ul>
+
+        <div class="col-md-12 col-sm-12 col-xs-12 truncate-ellipsis">
+          <div class="col-md-8 col-sm-7 col-xs-12 wi-detail-title-blk">
+            <div *ngIf="!headerEditable" (click)='openHeader()'
+              id="title-click-div">
+                <h2 *ngIf="!loggedIn" id="wi-detail-title-ne" class="detail-title truncate">{{ workItem.attributes['system.title'] }}</h2>
+                <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{ workItem.attributes['system.title'] }}</h2>
+                <span *ngIf="loggedIn && workItem.id"
+                  class="pficon-edit marginL10 fl detail-title-edit-ico"
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html]: indent it with ngIf

Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html]: Kya matlab?

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss (hunk 4 above)
Comment (nimishamukherjee) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: Ok

**Comment by nimishamukherjee** [comment]
[test]

**Comment by sanbornsen** [comment]
[test]

**Comment by sanbornsen** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Pull request merged**